src/data-access/apiBasedItemSearcher.ts

```diff
@@ -0,0 +1,47 @@
+import { SearchedItemOption } from '@/data-access/ItemSearcher';
+
+/*
+ * Utility code shared by several API-based ItemSearcher implementations.
+ */
+
+export const commonParams: Record<string, string> = {
```

Comment:
Not sure if these are worth it, beyond the magic number. But not too harmful either, I guess.

Reply:
Yeah, it had felt like more common parameters when I was copying them :D I wouldn’t mind losing these either.